docs/index.md

```diff
@@ -44,6 +44,13 @@ Install Codacy on an existing cluster using our Helm chart:
 
     -   [Helm](https://helm.sh/docs/intro/install/) version 3.2
 
+        !!! warning
+            Helm 3.3 is not supported because there is a known incompatibility with the MinIO chart currently used by Codacy.
+            
+            More specifically, **Helm versions compiled with Go 1.14 or later are not supported**. You can check the Go version that was used to compile Helm with `helm version`.
+            
+            This will be fixed in a future version of Codacy Self-hosted.
+
```

Comment:
I updated the notice to state that this issue will be fixed in a next version of Codacy Self-hosted (I omitted that it would be fixed on the next version just so we're safe in case something goes wrong).

I also added the [detail about the Go version that causes the incompatibility](https://codacy.slack.com/archives/C8X9SS1H7/p1596830473089300?thread_ts=1596745044.073100&cid=C8X9SS1H7), as it might save a lot of time and trouble to some other customer that is unaware of this.